plugins/source/test/resources/services/testdata.go

```diff
@@ -13,7 +13,7 @@ func TestDataTable() *schema.Table {
 		if strings.HasPrefix(c.Name, "_cq_") {
 			table.Columns[i].Name = "test" + c.Name
 		}
-		table.Columns[i].PrimaryKey = false
+		table.Columns[i].PrimaryKey = c.Name == "int64"
```

Comment:
This was a wild moment to test something, should I remove it so that we have a table with no PKs?

Reply:
IMO it's good to have a table with no PKs in test source until the moment we ban them completely